### DISCUSSION TITLE
Is it possible to integrate own, local fonts?

### DISCUSSION THREAD

**Discussion by friedemannm** [google/docsy#1015] [Q&A]
After extensive research, I have unfortunately found no reference to it.It would be very important for me to be able to use the CI font.

Thank you very much!

**Comment by fekete-robert** [comment]
Hi, yes, see https://www.docsy.dev/docs/adding-content/lookandfeel/#fonts

**Reply by friedemannm** [reply]
Thank you very much for the quick response. But there is no hint about how to integrate a .otf or .ttf file... Or I'm blind and I just didn't See it? 🤔

**Reply by fekete-robert** [reply]
Oh, I understand. AFAIK,  just:
- disable the variable mentioned in the docs
- copy the fonts into the /static/fonts folder of your project, then 
- reference it in your _styles_project.scss file, like:
    ```@font-face {
	font-family: "MyFontExtraLight";
	src: url("/fonts/MyFontExtraLight.ttf");
	font-weight: 200;
	font-style: normal;
	font-display: swap;
    }
    ```
